cmd/spell.go

```diff
@@ -51,6 +51,10 @@ func spellFunc(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	// TODO: if any validator is not yet a validator on the primary network,
```

Comment:
We should always do the min stake amount on the network and do for the duration specified in the command